specs/phase1/validator.md

```diff
@@ -275,6 +275,8 @@ Set `attestation_data.shard_head_root = hash_tree_root(shard_head_block)`.
 
 Set `shard_transition` to the value returned by `get_shard_transition(head_state, shard, shard_blocks)`.
 
+`get_block_data_merkle_root(data: ByteList) -> Root` is the function that returns the Merkle root of the block data without the length mixing.
```

Comment:
Let's elevate this definition to the phase1 beacon or custody spec. The same remerkleable call is currently used in the custody spec here -- https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase1/custody-game.md#custody-slashings